packages/extension/src/newtab/ShortcutLinks/useShortcutLinks.tsx

```diff
@@ -133,49 +142,33 @@ export default function useShortcutLinks(): UseShortcutLinks {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isManual, hasCustomLinks]);
 
-  return useMemo(
-    () => ({
-      formRef,
-      isManual,
-      formLinks,
-      shortcutLinks,
-      hasTopSites,
-      isTopSiteActive,
-      hasCheckedPermission,
-      customLinks,
-      onSaveChanges,
-      askTopSitesPermission: async () => {
-        const granted = await askTopSitesPermission();
-
-        if (granted) {
-          logEvent({
-            event_name: LogEvent.SaveShortcutAccess,
-            target_type: TargetType.Shortcuts,
-            extra: JSON.stringify({ source: ShortcutsSourceType.Browser }),
-          });
-        }
-
-        return granted;
-      },
-      resetSelected,
-      onIsManual: setIsManual,
-      revokePermission: onRevokePermission,
-    }),
-    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [
-      formRef,
-      isManual,
-      formLinks,
-      shortcutLinks,
-      hasTopSites,
-      isTopSiteActive,
-      hasCheckedPermission,
-      onSaveChanges,
-      askTopSitesPermission,
-      resetSelected,
-      setIsManual,
-      onRevokePermission,
-    ],
-  );
+  return {
+    formRef,
+    isManual,
+    formLinks,
+    shortcutLinks,
+    hasTopSites,
+    isTopSiteActive,
+    hasCheckedPermission,
+    customLinks,
+    showGetStarted,
+    hideShortcuts: showToggleShortcuts,
+    onSaveChanges,
+    askTopSitesPermission: async () => {
```

Comment:
Can't remember if this was used in any dependencies, but should we maybe wrap it in a useCallback to make the returned function stable?

Reply:
Well, I've seen most of the functions, are not wrapped, probably good to check whether we should add the wrap.